src/components/procedure/_remove-ruleset.mdx

```diff
@@ -1,20 +1,10 @@
-To **remove a ruleset** from the Rule board:
+To **disable a ruleset** from the Policies page:
 
-1. In Semgrep Cloud Platform, click **[Rule board](https://semgrep.dev/orgs/-/board)**.
-2. Click the ruleset that contains the rule.
-3. Click the <i className="fa-solid fa-trash-can inline_svg"></i> **Remove ruleset** icon next to the rule you're deleting.
-4. Click **Save**.
+1. In Semgrep Cloud Platform, click **Rules > [Policies](https://semgrep.dev/orgs/-/policies)**.
+2. From the **Ruleset** drop-down box, click the ruleset to remove.
+3. Click the <i class="fas fa-check-square"></i> **Matching rules**.
+4. Click **Edit > Disabled**.
 
 :::info
-- Individual rules within rulesets can only be disabled, not removed from the Rule board. To disable an individual rule in **[Rule board](https://semgrep.dev/orgs/-/board)**, click the <i class="fa-solid fa-toggle-large-on"></i> toggle to disable the rule. See also [Disabling rules](/semgrep-code/rule-board/#disabling-rules).
-- When you remove a rule from the Rule Board, all associated findings on Findings page and Dashboard page are removed also.
+- When you disable a rule, all previous findings generated by that rule are still retained.
 :::
-
-You can also remove a rule on the Findings page, to do so, follow these steps:
```

Comment:
This is still possible if you don't use group-by-rule - is there a reason it was removed?

Reply:
Thanks for catching that. I'll update the docs here :)